src/ccs/confluent.sh

```diff
@@ -993,6 +993,18 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    if [[ "x${platform}" = "xDarwin" ]]; then
+        local mac_version=$(sw_vers -productVersion)
+        local mac_version_major=$(echo ${mac_version} | cut -f 1 -d '.')
+        local mac_version_minor=$(echo ${mac_version} | cut -f 2 -d '.')
+        if [[ ${mac_version_major} -lt 10 || ( ${mac_version_major} -ge 10 && ${mac_version_minor} -lt 13 ) ]]; then
+            die "ERROR: macOS version >= 10.13 is required for all Confluent Platform features to work (detected macOS: ${mac_version})."
```

Comment:
nit: Let's keep error messages' format same with `Error:` vs `ERROR`. Warning was an outlier that was added with java version check.

Reply:
Ah my bad -- done